### ISSUE TITLE
Null pointer when using a generic lexer builder with a String token

### ISSUE THREAD

**Issue by StijnOostdam** [b3b00/csly#261]
The enum:

```
enum TestToken
{		
	[String]
	test = 1
}
```

The builder:

```
var buildResult = LexerBuilder.BuildLexer<MethodToken>();
```

This throws a null pointer exception. It works when I change the token to:

```
enum TestToken
{		
	[Lexeme(GenericToken.String, "\"", "\\")]
	test = 1
}
```

However, the documentation (https://github.com/b3b00/csly/wiki/GenericLexer) states that there should be default values for both the delimiter and escape delimiter, so I expect the first enum to work and be equal to the second enum.

**Comment by b3b00** [comment]
Will look at it rela soon

**Comment by StijnOostdam** [comment]
Thanks, don't worry, I can use the second enum defintion just fine

**Comment by b3b00** [comment]
fixed with release 2.8.0.2

**Comment by StijnOostdam** [comment]
Thank you!
